scrapy/commands/check.py

```diff
@@ -67,27 +71,57 @@ def run(self, args, opts):
         contract_reqs = defaultdict(list)
 
         spider_loader = self.crawler_process.spider_loader
-
+        
+        b1 = false
+        b2 = false
+        b3 = false
+        b4 = false
         for spidername in args or spider_loader.list():
+            b1 = true
             spidercls = spider_loader.load(spidername)
             spidercls.start_requests = lambda s: conman.from_spider(s, result)
 
             tested_methods = conman.tested_methods_from_spidercls(spidercls)
             if opts.list:
+                b2 = true
                 for method in tested_methods:
+                    b3 = true
                     contract_reqs[spidercls.name].append(method)
             elif tested_methods:
+                b4 = true
                 self.crawler_process.crawl(spidercls)
+        if(b1):
+            f.write("0001\n")
+        if(b2):
+            f.write("0002\n")
+        if(b3):
+            f.write("0003\n")
+        if(b4):
+            f.write("0004\n")
 
         # start checks
         if opts.list:
+            f.write("0005\n")
+            b1 = false
+            b2 = false
```

Comment:
probably redo this to just write to the file instead as u did for the other functions